app/lib/actions/getPharmacy.js

```diff
@@ -0,0 +1,18 @@
+const apiRequest = require('../etl-toolkit/apiRequest');
+const fixPhoneNumbers = require('../fixPhoneNumbers');
+const config = require('../config');
+
+function toObject(pharmaString, odsCode) {
+  const pharmacy = JSON.parse(pharmaString);
+  // eslint-disable-next-line no-underscore-dangle
+  pharmacy._id = odsCode;
+  fixPhoneNumbers(pharmacy);
```

Comment:
I mistyped here and meant to  reference the combiner component rather than the updater component. The reason for not moving it out just yet is fine. Perhaps add an issue to so that there is visibility of the work to do to move this out.